fdbserver/VersionedBTree.actor.cpp

```diff
@@ -9224,8 +9403,17 @@ TEST_CASE("Lredwood/correctness/btree") {
 	state int concurrentExtentReads =
 	    params.getInt("concurrentExtentReads").orDefault(SERVER_KNOBS->REDWOOD_EXTENT_CONCURRENT_READS);
 
+	state EncodingType encodingType = EncodingType::XXHash64;
+	state std::unique_ptr<IEncryptionKeyProvider> pKeyProvider = nullptr;
+
+	if (deterministicRandom()->coinflip()) {
```

Comment:
ditto

Reply:
Not sure what you mean?

Reply:
Comment is same as for line 7482. 

My suggestion is better to have a KNOB as a master switch to choose if a page gets encrypted; for now you have code to deterministically determine when to encrypt, my point is in future with such a SWITCH we can better control when a workload gets encrypted. 
Does it make sense?

Reply:
Oh, the knob.  There isn't any reason for the correctness unit test to not test encryption as part of its random parameter selection.  If there was, we could add a unit test parameter, read from `params` as with the other options which restrict the testing surface that are set above this line.